docs/usage/config-presets.md

```diff
@@ -180,11 +182,22 @@ To host your preset config on GitLab:
 - Add a renovate.json to this new repo containing the preset config. No other files are necessary
 - In other repos, reference it in an extends array like "gitlab>owner/name", e.g. "gitlab>rarkins/renovate-config"
 
+## Gitea-hosted Presets
+
+It is also possible to host your preset config using just a regular Gitea repository and without needing to publish it to npmjs.
+In such cases Renovate will simply look for a renovate.json file in the default branch, (for now only the master branch is supported).
+
+To host your preset config on Gitea:
+
+- Create a new repository on Gitea. Normally you'd call it renovate-config but it can be named anything
+- Add a renovate.json to this new repo containing the preset config. No other files are necessary
+- In other repos, reference it in an extends array like "gitea>owner/name", e.g. "gitea>rarkins/renovate-config"
+
```

Comment:
```suggestion
## Gitea-hosted Presets

It is also possible to host your preset config using just a regular Gitea repository and without needing to publish it to npmjs.
In such cases Renovate will simply look for a `renovate.json` file in the default branch, (for now only the _master_ branch is supported).

To host your preset config on Gitea:

- Create a new repository on Gitea. Normally you'd call it `renovate-config` but you can use any name you want
- Add a `renovate.json` to this new repository containing the preset config. No other files are necessary
- In other repositories, reference it in an extends array like `"gitea>owner/name"`, e.g. `"gitea>rarkins/renovate-config"`

```

## Changes:

- Add monospaced fonts where we're talking about files.
- Italicize word _master_
- Spell out word repository/repositories fully
- Simplify end of sentence: `but it can be named anything` -> `you can use any name you want`

Reply:
Thanks, maybe you can have a view on those files again after we merge this. I've only copied the github section 🙃